src/components/List/CheckBoxList.tsx

```diff
@@ -1,15 +1,19 @@
 'use client';
 
+import { CommonCheckBox } from '@/components/CheckBox/CommonCheckBox';
 import type { PrefectureData } from '@/types/population';
 import { fetchPrefectures } from '@/utils/population/fetchPrefectures';
 import { useEffect, useState } from 'react';
-import { CheckBox } from '../CheckBoxes/CheckBox';
 
-export const CheckBoxList = () => {
+type CheckBoxListProps = {
+    onChange: (selectedPrefectures: PrefectureData[]) => void; // 選択された都道府県データ
+};
```

Comment:
```suggestion
type CheckBoxListProps = {
    // チェック状態が変更された場合に呼び出される関数
    // 親コンポーネントに選択された都道府県データの配列を渡す
    onChange: (selectedPrefectures: PrefectureData[]) => void;
};
```
こんな感じ？
少なくとも onChange は関数だから、選択された都道府県データの配列そのものではない。

Reply:
対応した。対応したので、resolve する。